src/tests/system/tests/test_passkey.py

```diff
@@ -3,11 +3,67 @@
 
 :requirement: passkey
 
+Passkeys allow users to authenticate without having to enter a username or password,
+or provide any additional authentication factor.
+This technology aims to replace legacy authentication mechanisms such as passwords.
+
+The objective is to use a passkey to locally authenticate a user against centralized identity management system.
+For that purpose an integration with a server like AD/Samba or IPA or LDAP is needed.
+
 The passkey solution only enables to authenticate in a system where the
-FIDO2 key is connected physically.This could be su login, the GUI,
+FIDO2 key is connected physically.This could be su login, the GDM login,
 or even ssh<user>@localhost.
```

Comment:
even SSH. I don't think @localhost is necessary, in theory, if the passkey was added on the remote host, it would allow you to login correct?